### PR TITLE
Customization changes now update immediately

### PR DESCRIPTION
**Pull request by benfrancis314**
When you click it, it changes in live time instead of waiting for the next sentence. 

NOTE: This is based on the branch from my most recent PR, not master. This is bc it relies on some of the new class architecture developed in that PR. As such, if you look at this now, you will see those changes in addition to the ones solely belonging to this PR. For that reason, it may be best to look at this once that PR is merged with master.

### CODE REVIEW

File: extension/content_script.js [hunk 1]
```diff
@@ -264,10 +263,9 @@ function highlight(tracker) {
     	length: end - start
 	}], {
 		// "trackerClass" is for finding current tracker
-		className: sentenceClass
+		className: sentenceClass+" "+sentenceStyle
```
Comment (benfrancis314) [inline extension/content_script.js:266]: This adds both classes at once

File: extension/lib/settings_view.js [hunk 2]
```diff
@@ -26,10 +26,14 @@ class TrackerStyle {
         return this.keywordStyle;
     }
     setSentenceStyle(sentenceStyle) {
+        let oldSentenceStyle = this.sentenceStyle;
         this.sentenceStyle = sentenceStyle;
+        $("."+oldSentenceStyle).removeClass(oldSentenceStyle).addClass(sentenceStyle);;
```
Comment (sjonany) [inline extension/lib/settings_view.js]: nit: double semicolon

File: extension/content_script.js (hunk 1 above)
Comment (sjonany) [reply extension/content_script.js:266]: oo nice find! I didn't know you can do this

**Review by sjonany — APPROVED** [review]
Just one nit. Nice fix!

**Pull request merged**